modulo_lib_test.cc

```diff
@@ -0,0 +1,55 @@
+#include "modulo_lib.h"
+#include <iostream>
+
+#define ASSERT_EQUAL(a, b) \
+if (a != b) { \
+	std::cout << "   " << __FILE__ << ":" << __LINE__ \
+	          << " Expectation not met: " << a << " == " << b \
+	          << std::endl; \
+	return false; \
+}
+
+// Returns ANSI colorized message.
+std::string color(int color_index, const std::string& message) {
+	std::string codes[] = {
+		"\033[1;32m", // Bold Green
+		"\033[1;31m", // Bold Red
+	};
+	return codes[color_index] + message + "\033[0m";
+}
+
+// Returns ANSI colorized test status.
+std::string status(bool passed) {
+	if (passed) {
+		return color(0, "PASS");
+	}
+	return color(1, "FAIL");
+}
+
+
+bool test_modulo_multiply() {
+    // TODO(tomichaw): Implement this.
+	return true;
+}
+
+bool test_modulo_add() {
+    // TODO(tomichaw): Implement this.
+	return true;
+}
+
+// QUESTION: How do you test the "print" function?
```

Comment:
The print function is not testable as it is. Maybe we should pass an ostream object into the print() function which it then uses to output the formatted message? That way, we can pass in an ostringstream from the test and check its content.

Reply:
Great idea! I will add a feature request.